iam/src/providers/githubFollowers.ts

```diff
@@ -41,7 +41,7 @@ export class TenOrMoreGithubFollowers implements Provider {
       return { valid: false };
     } finally {
       valid =
-        verifiedPayload && verifiedPayload.followers >= 10 && verifiedPayload.followers > 50 && verifiedPayload.id
+        verifiedPayload && verifiedPayload.followers >= 10 && verifiedPayload.followers < 50 && verifiedPayload.id
```

Comment:
Should we be capping this with a max? It might be better to just leave it open, if the user has more than 50 followers, they also have more than 10 🤔

Reply:
Just pushed these changes 💪